pages/common/php-cs-fixer.md

```diff
@@ -0,0 +1,36 @@
+# PHP-CS-Fixer
+
+> Automatic coding style fixer for PHP.
+> More information: <https://github.com/FriendsOfPHP/PHP-CS-Fixer>.
+
+- Execute code style fixing:
```

Comment:
I assume this is correct since the next example specifies a directory.
Better to add some clarity if so.
```suggestion
- Execute code style fixing in the current directory:
```

Reply:
Yeah, sounds good. 👍🏻 It's a bit weird as technically when you don't specify a path it uses directories from a config. So you might have a config that specifies only files in a certain subdirectory, even though you're running the command in the parent directory (where the config is).